api/server.py

```
@@ -0,0 +1,92 @@
+from fastapi import FastAPI, HTTPException
+from pydantic import BaseModel
+import joblib
+import pandas as pd
+from data_preprocessor import preprocess_test
+
+# Define input data model using Pydantic
+from pydantic import BaseModel, Field
```

Ruff (0.8.2)

Redefinition of unused `BaseModel` from line 2

Remove definition: `BaseModel`

(F811)